### DISCUSSION TITLE
Primefaces 14.0.2 - Autoupdate on growl doesn't work anymore

### DISCUSSION THREAD

**Discussion by AlessandroBori** [primefaces/community#2253] [PrimeFaces] [answered]
Hi everybody!
After upgrading from 12.0.5 to 14.0.2, following guidelines, if on a page the view has attached an event (like postaddtoview or prerendercomponent, it's the same) with a listener, autoupdating on growl messages works fine on the first ajax call then stops.

As you can see in the test project in attachment running on Java 11 Mojarra 2.3.

[primefaces-test-master.zip](https://github.com/user-attachments/files/16096138/primefaces-test-master.zip)

>>>> Useful Tip: it worked until 14.0.0 RC1

**Comment by melloware** [comment]
@AlessandroBori can you open a PF ticket please and be sure to include your reproducer.

**Reply by AlessandroBori** [reply]
ok

**Reply by melloware** [reply, marked as answer]
If I had to guess it was this change: https://github.com/primefaces/primefaces/issues/11408

New Ticket: https://github.com/primefaces/primefaces/issues/12247
